ros2/test.bzl

```diff
@@ -1,12 +1,12 @@
 """ Defines ROS2 testing functionality.
 """
 
-load("@bazel_skylib//lib:paths.bzl", "paths")
 load("@com_github_mvukov_rules_ros2//ros2:ament.bzl", "ros2_ament_setup")
 load("@com_github_mvukov_rules_ros2//third_party:expand_template.bzl", "expand_template")
+load("@com_github_mvukov_rules_ros2//third_party/pytest:defs.bzl", "py_pytest_test")
 load("@rules_python//python:defs.bzl", "py_test")
 
-def ros2_test(name, nodes, launch_file, deps = None, data = None, **kwargs):
+def ros2_test(name, nodes, launch_file, deps = [], data = [], use_pytest = False, **kwargs):
```

Comment:
It's quite a good practice to set default args as None (immutable), opposed to mutable args such as empty lists. Please revert.